go.sum

```diff
@@ -32,12 +32,6 @@ github.com/davecgh/go-spew v1.1.1/go.mod h1:J7Y8YcW2NihsgmVo/mv3lAwl/skON4iLHjSs
 github.com/dgrijalva/jwt-go v0.0.0-20160705203006-01aeca54ebda/go.mod h1:E3ru+11k8xSBh+hMPgOLZmtrrCbhqsmaPHjLKYnJCaQ=
 github.com/dgrijalva/jwt-go v3.2.0+incompatible/go.mod h1:E3ru+11k8xSBh+hMPgOLZmtrrCbhqsmaPHjLKYnJCaQ=
 github.com/dgryski/go-sip13 v0.0.0-20181026042036-e10d5fee7954/go.mod h1:vAd38F8PWV+bWy6jNmig1y/TA+kYO4g3RSRF0IAv0no=
-github.com/digitalocean/godo v1.22.0 h1:bVFBKXW2TlynZ9SqmlM6ZSW6UPEzFckltSIUT5NC8L4=
-github.com/digitalocean/godo v1.22.0/go.mod h1:iJnN9rVu6K5LioLxLimlq0uRI+y/eAQjROUmeU/r0hY=
-github.com/digitalocean/godo v1.24.2-0.20191106222530-9ccfbd9c44ff h1:/tA+xwBHCwWdwizotGgjXrosBupethua3SMQ9Rozyrg=
```

Comment:
this is interesting that the sum changed? I guess we didn't run a tidy and all this was left over from other pulls?

Reply:
cc @bentranter

Reply:
yeah I just ran a tidy on that one and got this after rebase with master

Reply:
@gregmankes - thanks for that. We should have double checked that on the other PR.